styleguide/_themes/derek/scss/components/_basement.scss

```diff
@@ -7,12 +7,12 @@
 
 // Basement Classes
 .basement {
-  padding: 10px 15px;
+  @include make-container;
+  padding: 10px 0;
 }
 
-.basement-copyright {
-  color: $medium-gray;
-  text-align: right;
+.basement-row {
+  @extend %row;
```

Comment:
This should probably just be `@include make-row;` not `@extend %row;` as that's all it's doing anyway. Seems like we're adding an extra step here by using and extend